src/content/api.jl

```diff
@@ -1,9 +1,7 @@
 export body!, content!, loadcss!, loadjs!, load!, importhtml!
 
 content!(o, sel, html::AbstractString; fade = true) =
-  fade ?
-    @js_(o, Blink.fill($sel, $html)) :
-    @js_ o document.querySelector($sel).innerHTML = $html
+    @js_(o, Blink.fill($sel, $html, $(fade ? 1 : 0)))
```

Comment:
Is the `fade ? 1 : 0` really necessary?

Reply:
Oops, no, it's definitely not. Thanks. I think I was worried that javascript doesn't have values for `true` or `false`, but that's silly. XD
Fixed in the rebased version i just pushed! :) thanks!